sdk/core/Azure.Core/README.md

```diff
@@ -1,45 +1,100 @@
-        message.SetRequestLine(HttpVerb.Get, uri);
-
-        // add headers
-        message.AddHeader("Host", uri.Host);
-
-        // send message
-        await pipeline.SendMessageAsync(message).ConfigureAwait(false);
-
-        // process response
-        Response response = message.Response;
-        if (response.Status == 200) {
-            var reader = new StreamReader(response.ContentStream);
-            string responseText = reader.ReadToEnd();
-        }
-        else throw new RequestFailedException(response);
-    }       
+    // BlobConnectionOptions inherits/extends ClientOptions
+    ClientOptions options = new BlobConnectionOptions();     
+    
+    // configure retries
+    options.RetryPolicy.MaxRetries = 5; // default is 3
+    options.RetryPolicy.Mode = RetryMode.Exponential; // default is fixed retry policy
+    options.RetryPolicy.Delay = TimeSpan.FromSeconds(1); // derfault is 0.8s
+
+    // finally create BlobContainerClient, but many Azure SDK clients will work similarly
+    var client = new BlobContainerClient(connectionString, "container", options);
 }
 ```
 
-# Other Samples
+## Accessing HTTP Response Details Using ```Response<T>```
+_Service clients_ have methods that can be used to call Azure services. 
+We refer to these client methods _service methods_.
+_Service methods_ return a shared Azure.Core type ```Response<T>``` (in rare cases its non-generic sibling ```Response```).
+This type provides access to both the deserialized result of the service call, 
+and to the details of the HTTP response returned from the server.
+
+```csharp
+public async Task UsingResponseOfT()
+{
+    // create a client
+    var client = new BlobContainerClient(connectionString, "container");
+
+    // call a service method, which returns Response<T>
+    Response<ContainerItem> response = await client.GetPropertiesAsync();
+
+    // Response<T> has two main accessors. 
+    // Value property for accessing the deserialized result of the call
+    ContainerItem container = response.Value;
+
+    // .. and GetRawResponse method for accessing all the details of the HTTP response
+    Response http = response.GetRawResponse();
 
+    // for example, you can access HTTP status
+    int status = http.Status;
+
+    // or the headers
+    foreach(HttpHeader header = http.Headers) {
+        Console.WriteLine($"{header.Name} {header.Value}");
+    }
+
+    // or the stream of the response content
+    Stream content = http.ContentStream;
+
+    // but, if you are not interested in all HTTP details, 
+    // and just want the result of the service call,
+    // Response<T> provides a cast to get you directly to the result
+    ContainerItem result = await client.GetPropertiesAsync();
+}
+```
+
+## Consuming Service Methods Returning ```IAsyncEnumerable<T>```
+Coming soon ...
+
+## Consuming Long Running Operations Using ```OperationT<T>```
 Comming soon ...
+
+# Installing
```

Comment:
I would push this entire section to the very beginning - knowing that this is not something that someone actually usually needs to use is crucial

Reply:
OK, I moved the "installing" section to right after the overview.